spec/controllers/hosts_controller_spec.rb

```diff
@@ -12,7 +12,7 @@
     let(:host_email) { 'johnnyenglish@mrbean.com' }
     let(:params) do
       {
-        name: 'Mr. Bean',
+        nickname: 'Mr. Bean',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.